aspnetcore/performance/caching/middleware.md

```diff
@@ -1,15 +1,104 @@
 ---
-title: 🔧 Response caching middleware
+title: Response Caching Middleware | Microsoft Docs
 author: rick-anderson
 ms.author: riande
 manager: wpickett
-ms.date: 10/14/2016
+ms.date: 12/05/2016
 ms.topic: article
 ms.assetid: f9267eab-2762-42ac-1638-4a25d2c9d67c
 ms.prod: aspnet-core
 uid: performance/caching/middleware
 ---
-# 🔧 Response caching middleware
+# Response Caching Middleware
+
+By [Luke Latham](https://github.com/GuardRex) and [John Luo](https://github.com/JunTaoLuo)
+
+[View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/performance/caching/response/sample)
+
+This document provides details on how to configure the Response Caching Middleware for ASP.NET Core applications. For an introduction to to HTTP caching and the `ResponseCaching` feature, see [Response Caching](response.md).
+
+## Package
+To include the middleware in your project, add a reference to the  [`Microsoft.AspNetCore.ResponseCaching`](https://www.nuget.org/packages/Microsoft.AspNetCore.ResponseCaching/) package. The middleware is available for projects that target `.NETFramework 4.5.1` or `.NETStandard 1.3` or higher.
+
+## Extensions
+In `ConfgureServices`, add the middleware to your service collection.
+```c#
```

Comment:
It seems variable to me ... e.g. ...
* https://docs.microsoft.com/en-us/aspnet/core/fundamentals/static-files
* https://docs.microsoft.com/en-us/aspnet/core/security/authentication/cookie
* https://docs.microsoft.com/en-us/aspnet/core/security/cors

They're not exactly consistent, except that they are showing some kind of small snippets with or without `Configure Services` and `Configure`. I could *definitely* drop those stupid ellipsis dots (What was I thinking?! lol). Do you want me to show the full `ConfigureServices` and `Configure` from the sample in these spots?